### ISSUE TITLE
Version 5.5 is causing bootlop

### ISSUE THREAD

**Issue by doclucas** [oasisfeng/island#247]
Installed the latest pre-release 5.5 on Galaxy Note 10 N975F running Android 11.
Created the island using root and rebooted.
Once the booting reaches the loging screen it restarts the UI constantly (loop), not a complete reboot, but still looping.

**Comment by oasisfeng** [comment]
Could you capture the logcat containing the "FATAL EXCEPTION" causing boot loop?

**Comment by doclucas** [comment]
Yeah, I probably should have done that, but unfortunately I have already nandroid restored the device...

**Comment by oasisfeng** [comment]
Have you tried creating Island space normally (without root)? Is it working properly?

**Comment by doclucas** [comment]
I used root because it wasn't able to create the work space without it.

**Comment by cyyself** [comment]
It seems like a long-standing issue, even on the latest Island with Android 16. Maybe `adb shell pm remove user ${WORK_PROFILE_ID}` would help. You can obtain `${WORK_PROFILE_ID}` by executing `adb shell pm list users`.

<img width="474" height="199" alt="Image" src="https://github.com/user-attachments/assets/330fa813-8c6f-411d-aea6-81eed02dc512" />

I have saved the data on my friend's phone with the same issue. She just created a work profile using island, cloned some apps, destroyed it, and then create again. Then, the phone reboots during creation, and then 2-4s after entering the password on the login screen, the phone reboots (only Android userspace, not linux kernel or system reboot).

And I finally managed to open the ADB in 2-4s operation window before reboot using USB HID Emulation to have ADB open and remove the work profile to make the phone back to normal. If anyone needs such help to save data in a bootloop, my blog would help: https://blog.cyyself.name/usb-hid-emulation-fix-android-bootloop/

I recommend having ADB open before such operations to avoid such issues.

I also recommend renaming this issue to "boot loop" so that anyone facing such an issue can easily search for it and find the solution here.
